tdo.sh

```diff
@@ -173,14 +199,15 @@ new_entry() {
     entry_file="$root/entries/$(generate_file_path "$1")"
     template="$root/templates/entry.md"
     create_file "$entry_file" "$template"
-    add_timestamp "$entry_file"
+    [ $add_entry_timestamp = "true" ] && add_timestamp "$entry_file" "$entry_timestamp_format"
```

Comment:
can be simplified by eliminating string check
```suggestion
    $add_entry_timestamp && add_timestamp "$entry_file" "$entry_timestamp_format"
```